newrelic-alerts-configurator-dsl/src/main/kotlin/com/ocadotechnology/newrelic/alertsconfigurator/dsl/configuration/condition/NrqlConditionConfigurationDsl.kt

```diff
@@ -8,7 +8,7 @@ import com.ocadotechnology.newrelic.alertsconfigurator.dsl.NewRelicConfiguration
 @NewRelicConfigurationMarker
 class NrqlSignalConfigurationDsl {
     var aggregationWindow: Int? = null
-    var evaluationWindows: Int? = null
+    var aggregationDelay: Int? = null
```

Comment:
Don't we need aggregationMethod as well?

Reply:
Added